src/host/settings.cpp

```diff
@@ -404,13 +403,13 @@ void Settings::SetFilterOnPaste(const bool fFilterOnPaste)
     _fFilterOnPaste = fFilterOnPaste;
 }
 
-const WCHAR* const Settings::GetLaunchFaceName() const
+const std::wstring& Settings::GetLaunchFaceName() const

```

Comment:
What? Why? Why not just a std::wstring_view?

Reply:
/shrug no real reason, i went with your first suggestion instead of your second